packages/ts/src/components/grouped-bar/index.ts

```diff
@@ -322,4 +318,9 @@ export class GroupedBar<Datum> extends XYComponentCore<Datum, GroupedBarConfig<D
       ? this.getDataScaleExtent()
       : this.getValueScaleExtent(false)
   }
+
+  getAriaDescription (): string {
+    const description = super.getAriaDescription() || ''
```

Comment:
I'm not sure that calling `super.getAriaDescription()` is a good idea. If you have multiple components in your chart, you'll most  likely end up with having duplicated items in the description.

Reply:
I was under the impression that each XY-component will generate a unique description as they will have a unique x extent and y extent ?